### PR TITLE
chore: add yarn.lock in source control

### LINKED ISSUE #41: Adding a yarn.lock
According to yarn docs, the `yarn.lock` file should be checked into source control.
https://classic.yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control)
What do you think of this ?

### PR DESCRIPTION
**Pull request by GuillaumeOj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "url": "https://github.com/Mergifyio/react-crisp"
   },
   "jest": {
-    "setupFilesAfterEnv": ["./jest.setup.js"]
+    "setupFilesAfterEnv": [
+      "./jest.setup.js"
+    ]
```
Comment (GuillaumeOj) [inline package.json:20]: The indentation change when we run a `yarn` command which update `package.json`

**Comment by jd** [comment]
You might want to also set up Mergify for dependabot, no?

**Comment by jd** [comment]
Also add Fixes #41 somewhere :)

**Comment by GuillaumeOj** [comment]
> You might want to also set up Mergify for dependabot, no?

Oops, I've update the set up via the config editor: #45

**Review by jd — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Psoffie009) [reply package.json:20]: > The indentation change when we run a `yarn` command which update `package.json`

**Pull request merged**